docs/modules/user-guide/partials/proc_adding-a-volume-component-to-a-devfile.adoc

```diff
@@ -0,0 +1,55 @@
+[id="proc_adding-a-volume-component-to-a-devfile_{context}"]
+= Adding a volume component to a devfile
+
+[role="_abstract"]
+You can use a volume component to share files among container components and collaborate easily with other teams during the development process.
+
+.Prerequisites
+
+* xref:adding-schema-version-to-a-devfile.adoc[]
+* xref:adding-a-name-to-a-devfile.adoc[]
+* xref:adding-components-to-a-devfile.adoc[]
+
+.Procedure
+
+. Define a `volume` component in your devfile and specify a volume size for it.
++
+.A minimal `volume` component
+[source,yaml]
+----
+schemaVersion: 2.1.0
+metadata:
+    name: mydevfile
+components:
+  - name: mydevfile
+    volume:
+      size: 2Gi
+----
+
+. Store an ephemeral `volume` component for it to be persistent across restarts.
++
+.An ephemeral `volume` component
+
+[source,yaml]
+----
+schemaVersion: 2.1.0
+metadata:
+    name: mydevfile
+components:
+  - name: mydevfile
+    volume:
+      ephemeral: true
+      size: 200G
+----
+
+[NOTE]
+====
+ * Implementing the size of the `volume` component is dependent on the tools you use, and therefore, might be subject to the limitations of the tools.
+ * For more details, refer to the documentation for your tools.
```

Comment:
@abrennan89 it was originally one sentence. In that case, users had to drag the space bar to read the whole sentence within the note. So I thought i'd be easier to read as bullet lists, but I can put it back as a sentence if you think that's best :)

Reply:
I think it would read better, however this sounds more like maybe a CSS issue re scrolling. Might be best to open an issue / PR to fix that, since it would continue to be a problem with long lines in the future. Maybe turn off horizontal scrolling or have some wrapping, although it'll depend on how your site is put together :)

Do you have a link for where the current version of this lives so I can take a look?